website/content/docs/connect/cluster-peering/configuration.mdx

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
For api gateway, we name this file [`tech-specs`](https://consul-cyxww49ps-hashicorp.vercel.app/consul/docs/api-gateway/tech-specs)

I would avoid naming file names `configuration` in case we want to create a configuration reference docs/directory